packages/editor/src/components/block-inspector/index.js

```diff
@@ -28,30 +30,44 @@ const BlockInspector = ( { selectedBlock, blockType, count } ) => {
 		return <span className="editor-block-inspector__no-blocks">{ __( 'No block selected.' ) }</span>;
 	}
 
-	return [
-		<div className="editor-block-inspector__card" key="card">
-			<BlockIcon icon={ blockType.icon } showColors />
-			<div className="editor-block-inspector__card-content">
-				<div className="editor-block-inspector__card-title">{ blockType.title }</div>
-				<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+	return (
+		<Fragment>
+			<div className="editor-block-inspector__card">
+				<BlockIcon icon={ blockType.icon } showColors />
+				<div className="editor-block-inspector__card-content">
+					<div className="editor-block-inspector__card-title">{ blockType.title }</div>
+					<div className="editor-block-inspector__card-description">{ blockType.description }</div>
+				</div>
 			</div>
-		</div>,
-		<div key="inspector-controls"><InspectorControls.Slot /></div>,
-		<div key="inspector-advanced-controls">
-			<InspectorAdvancedControls.Slot>
-				{ ( fills ) => ! isEmpty( fills ) && (
+			<div><InspectorControls.Slot /></div>
+			{ !! blockType.styles && (
+				<div className="editor-block-inspector__styles">
```

Comment:
Minor: We put a class name inside `div` in here and inside `PanelBody` for the advanced controls. Is it possible to use a common approach? It would be nice to put it inside `PabelBody` as we probably will want to remove those wrapping `div` elements at some point. It's NOT a blocker :)